dotcom-rendering/src/components/Metrics.importable.tsx

```diff
@@ -106,7 +157,15 @@ export const Metrics = ({ commercialMetricsEnabled }: Props) => {
 					),
 				);
 		},
-		[abTestApi, adBlockerInUse, commercialMetricsEnabled],
+		[
+			abTestApi,
+			adBlockerInUse,
```

Comment:
To replicate the `useOnce` behaviour should we include a check for this alongside the other `undefined` checks?

```ts
if (isUndefined(adBlockerInUse)) { return; }
```

Previously I think `useOnce` would have waited to run until `adBlockerInUse` was defined?